### PR TITLE
Please review new Health test SDK samples in the vra-services-check folder 

### PR DESCRIPTION
**Pull request by dzilbermanvmw**
I have forked into a clean repo, added my changes under 'vra-services-check' folder and verified they indeed contain the changes suggested by the maintainer. 
Please review and let me know if any other changes needed.

### CODE REVIEW

File: vra-services-check/setup_dev.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+setup(name='vrealize-health-vra-dev',
+      version='1.0.0',
+      description='VMware vRealize Automation Post Upgrade Health Service Checks ',
+      author='VMware, Inc.',
+      author_email='dzilberman@vmware.com',
+      url='https://github.com/vmwaresamples/vra-health-samples',
+      install_requires=['pyvmomi'],
```
Comment (jeffwcook) [inline vra-services-check/setup_dev.py]: Are you using this package?

File: vra-services-check/src/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+""" 
+Copyright © 2018 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the “License”); you may not
```
Comment (jeffwcook) [inline vra-services-check/src/__init__.py]: This is using the Apache license.  I think you mean to be using the MIT license as you have elsewhere.

File: vra-services-check/setup_dev.py (hunk 1 above)
Comment (jeffwcook) [reply vra-services-check/setup_dev.py]: Use the MIT license.  You have two licenses in use in this project,

File: vra-services-check/src/run_vrahealthtestsuite.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+#import json
```
Comment (jeffwcook) [inline vra-services-check/src/run_vrahealthtestsuite.py]: Add a license header here.

File: vra-services-check/src/testsuite_module/vrahealthtestsuite.py [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+            return None
+
+    #end of get_registered_services function definition
+
+    #start Test suite method 'check_registered_services' definition 
+    @test_name("Check number of vRA Registered services")
+    @test_description("Checks that core vRA Services are Registered Post Upgrade")
+    @test_access_level(AccessLevel.NORMAL.value)
+    @test_severity(Severity.NORMAL.value)
+    @test_remediation("https://www.vmware.com/products/vrealize-automation.html")
```
Comment (jeffwcook) [inline vra-services-check/src/testsuite_module/vrahealthtestsuite.py]: Consider a more specific remediation link.  Perhaps to the services section of the documentation?

File: vra-services-check/src/testsuite_module/vrahealthtestsuite.py [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+    @test
+    def check_distinct_endpoints(self, vraaddress, vrausername, vrapassword, vratenant, numendpoints):
+        
+        endpointsJSON = self.get_distinct_endpoint_types(vraaddress, vrausername, vrapassword, vratenant)
+
+        if endpointsJSON:
+            #print("Asserting length of all Distinct VRA endpoints >= " + str(numendpoints) + " - " + str(len(endpointsJSON) >= int(numendpoints)))
+            assert len(endpointsJSON) >= int(numendpoints), "Number of Distinct vRA endpoints: " + str(len(endpointsJSON)) + " is less than expected: " + str(numendpoints)
```
Comment (jeffwcook) [inline vra-services-check/src/testsuite_module/vrahealthtestsuite.py:164]: Is there a way to display which endpoint is actually missing?

**Review by jeffwcook — CHANGES_REQUESTED** [review]

**Review by jeffwcook — CHANGES_REQUESTED** [review]
Please remove the dist directory.   Don't check in binary files or build artifacts.

File: vra-services-check/src/testsuite_module/vrahealthtestsuite.py (hunk 5 above)
Comment (dzilberman) [reply vra-services-check/src/testsuite_module/vrahealthtestsuite.py:164]: Not at the moment as I'm just counting the endpoints and comparing against the target count I can work on adding that later

File: vra-services-check/setup_dev.py (hunk 1 above)
Comment (dzilberman) [reply vra-services-check/setup_dev.py]: I don't think I am using pyvmomi package, so will remove the requirement. Will use 'The MIT license (the “License”)..' in setup config/script files

File: vra-services-check/src/__init__.py (hunk 2 above)
Comment (dzilberman) [reply vra-services-check/src/__init__.py]: Yes, will use The MIT license (the “License”)

File: vra-services-check/src/testsuite_module/vrahealthtestsuite.py (hunk 4 above)
Comment (dzilberman) [reply vra-services-check/src/testsuite_module/vrahealthtestsuite.py]: @test_remediation("https://www.vmware.com/products/vrealize-automation.html") used for each of test cases to point to specific area of vRA 7.x documentation? Should I use 7.5 docs?

**Review by dzilberman — APPROVED** [review]
Changes approved

File: vra-services-check/src/run_vrahealthtestsuite.py (hunk 3 above)
Comment (dzilberman) [reply vra-services-check/src/run_vrahealthtestsuite.py]: I do have '# Copyright 2018 VMware, Inc.  All rights reserved				
# The MIT license (the â€œLicenseâ€�) set forth below ...
at the top of the file -  is that not sufficient? thx

Comment (jeffwcook) [reply vra-services-check/src/run_vrahealthtestsuite.py]: Every source file needs the license and the copyright.

File: vra-services-check/src/testsuite_module/vrahealthtestsuite.py (hunk 4 above)
Comment (jeffwcook) [reply vra-services-check/src/testsuite_module/vrahealthtestsuite.py]: yes, you'll have to use the current version (e.g. 7.5)

File: vra-services-check/src/testsuite_module/vrahealthtestsuite.py (hunk 5 above)
Comment (jeffwcook) [reply vra-services-check/src/testsuite_module/vrahealthtestsuite.py:164]: ok

**Review by jeffwcook — APPROVED** [review]

**Pull request merged**